src/components/Card/style.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../scss/variables/colors';
+
+$base-class: 'card';
+
+.#{$base-class} {
+  background-color: #fff;
+  color: $font-primary-color;
+  border: solid 1px $divider-color;
+  border-radius: 8px;
+  margin-top: 8px;
+  padding: 18px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 16px;
+  overflow: hidden;
+  
+  &__empty {
+    display: flex;
+    align-items: center;
+    margin-bottom: 10px;
+  }
+
+  &__title {
+    display: flex;
+    align-items: center;
+    margin-bottom: 14px;
+  }
+
+  &__text {
+    font-size: 18px;
+    line-height: 24px;
+    color: $ui-tooltip-color;
+    font-weight: 600;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 100%;
+  }
+
+  &__img {
+    height: 24px;
+    width: 24px;
+    margin-right: 10px;
+    border-radius: 4px;
+    display: inline-block;
+  }
+
+  &:last-of-type {
+    margin-bottom: 10px;
```

Comment:
hmm, could you elaborate on this?

Reply:
tbh it's copy paste syndrome 😅

Reply:
done at e57e61ea1e6ba66aa5c16156a890aba32e1c1e42